config/params.php

```diff
@@ -11,4 +11,10 @@
         'allowedIPs' => ['127.0.0.1', '::1'],
         'allowedHosts' => [],
     ],
+    'yiisoft/yii-swagger' => [
+        'annotation-paths' => [
```

Comment:
That should be sucessfully merged with https://github.com/yiisoft/demo-api/blob/master/config/params.php#L127, right?

Reply:
Yes.